docs(ja): translate context (#920)
nuxt/nuxtjs.org: content/ja/guides/internals-glossary/context.md

Vuex の Store インスタンス。**[Vuex ストア](/guide/vuex-store) が設定されている場合のみ使用できます**。

### route

`route` ([_Vue Router Route_](https://router.vuejs.org/ja/api/#%E3%83%AB%E3%83%BC%E3%83%88%E3%82%AA%E3%83%96%E3%82%B8%E3%82%A7%E3%82%AF%E3%83%88))

Vue Router の route インスタンス。

### params

`params` (_Object_)

`route.params` のエイリアス。

### query

`query` (_Object_)

`route.query` のエイリアス。

### env

`env` (_Object_)

`nuxt.config.js` で設定された環境変数。[env api](/guides/configuration-glossary/configuration-env) を参照してください。

### isDev

`isDev` (_Boolean_)

開発モードであるかどうかを示すブール値で、本番環境で一部のデータをキャッシュするのに役立ちます。

### isHMR

`isHMR` (_Boolean_)

メソッド/ミドルウェアが webpack の hot module replacement から呼び出されたかどうかを示すブール値（開発モードのクライアントサイドでのみ true）。

### redirect

`redirect` (_Function_)

ユーザーを別のルートにリダイレクトするには、このメソッドを使用してください。ステータスコードはサーバーサイドで使用され、デフォルトは `302` です。 `redirect([status,] path [, query])`。

例:

```js{}[]
redirect(302, '/login')
redirect({ name: 'slug', params: { slug: mySlug } })
redirect('https://vuejs.org')
```

Location プロパティの詳細については、[Vue Router のドキュメント](https://github.com/vuejs/vue-router/blob/64d60c01920405f0b93e00a401c73868b08ee6e5/types/router.d.ts#L161-L169)を参照してください。

<base-alert type="info">

> ハイドレーションエラー（クライアントの内容が、サーバーが返すものと異なってしまう）のため、[クライアントサイド限定の Nuxt プラグイン](/guides/directory-structure/plugins#client-or-server-side-only) の中では `redirect` や `error` を使用することはできません。
> 
> 有効な回避策は、 `window.onNuxtReady(() => { window.$nuxt.$router.push('/your-route') })` を使用することです。

Reviewer: ```suggestion
ハイドレーションエラー（クライアントの内容が、サーバーが返すものと異なってしまう）のため、[クライアントサイド限定の Nuxt プラグイン](/docs/2.x/directory-structure/plugins#client-or-server-side-only)の中では `redirect` や `error` を使用することはできません。

有効な回避策は、`window.onNuxtReady(() => { window.$nuxt.$router.push('/your-route') })` を使用することです。
```

</base-alert>

### error

`error` (_Function_)

> エラーページを表示するには、このメソッドを使用してください。 例: `error(params)`。 `params` には `statusCode` と `message` プロパティが必要です。

Reviewer: ```suggestion
エラーページを表示するには、このメソッドを使用してください。例: `error(params)`。`params` には `statusCode` と `message` プロパティが必要です。
```

### `$config`

`$config` (_Object_)

> 実際の [ランタイム設定](/guide/runtime-config)。

Reviewer: ```suggestion
実際の[ランタイム設定](/guide/runtime-config)。
```

## サーバーサイドのキー

これらのキーはサーバーサイドでのみ使用できます。

### req

> `req` ([_http.Request_](https://nodejs.org/api/http.html#http_class_http_incomingmessage))

Reviewer: ```suggestion
`req`（[_http.Request_](https://nodejs.org/api/http.html#http_class_http_incomingmessage)）
```

Node.js サーバーからのリクエスト。Nuxt がミドルウェアとして使われる場合、リクエストオブジェクトは使用しているフレームワークによって異なる可能性があります。<br>**`nuxt generate` では使用できません**。

### Res

> `res` ([_http.Response_](https://nodejs.org/api/http.html#http_class_http_serverresponse))

Reviewer: ```suggestion
`res`（[_http.Response_](https://nodejs.org/api/http.html#http_class_http_serverresponse)）
```

Node.js サーバーからのレスポンス。Nuxt がミドルウェアとして使われる場合、レスポンスオブジェクトは使用しているフレームワークによって異なる可能性があります。<br>**`nuxt generate` では使用できません**。

### beforeNuxtRender

`beforeNuxtRender(fn)` (_Function_)

> クライアントサイドでレンダリングされた `__NUXT__` 変数を更新するには、このメソッドを使用してください。`fn`（非同期にもできます）は `{ Components, nuxtState }` で呼び出されます。[例](https://github.com/nuxt/nuxt.js/blob/cf6b0df45f678c5ac35535d49710c606ab34787d/test/fixtures/basic/pages/special-state.vue) を参照してください。

Reviewer: ```suggestion
クライアントサイドでレンダリングされた `__NUXT__` 変数を更新するには、このメソッドを使用してください。`fn`（非同期にもできます）は `{ Components, nuxtState }` で呼び出されます。[例](https://github.com/nuxt/nuxt.js/blob/cf6b0df45f678c5ac35535d49710c606ab34787d/test/fixtures/basic/pages/special-state.vue)を参照してください。
```

## クライアントサイドのキー

これらのキーはクライアントサイドでのみ使用できます。

### from

> `from` ([_Vue Router Route_](https://router.vuejs.org/ja/api/#%E3%83%AB%E3%83%BC%E3%83%88%E3%82%AA%E3%83%96%E3%82%B8%E3%82%A7%E3%82%AF%E3%83%88))

Reviewer: ```suggestion
`from`（[_Vue Router Route_](https://router.vuejs.org/ja/api/#%E3%83%AB%E3%83%BC%E3%83%88%E3%82%AA%E3%83%96%E3%82%B8%E3%82%A7%E3%82%AF%E3%83%88)）
```
